### PR TITLE
Configuring Works & About Sections

### PR DESCRIPTION
**Pull request by canon20paul**
**************************************************************************************************************************
##  📝 PR Outline  📝

- **Works section** created. 
-*Mapping gride for Works & about
-Styling works Project*
- **About Section** created.
- Creation of Header and detail Area
- *Styling Area*

- **Languages Area** added
- **frameworks Area** added
- **Skills Area** added
- **Readme** UPDATED.
**************************************************************************************************************************

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -80,32 +80,137 @@ <h1 class="header1">Hey there. I'm &nbsp;&nbsp;&nbsp;&nbsp;Mariane&nbsp;&nbsp;&n
...
+                    <li class="lang-list-item">css</li>
+                    <li class="lang-list-item">Javascript</li>
+                    <li class="lang-list-item">html</li>
+                </ul>
+                <button type="submit" class="buttonW">See Project</button>
+            </div>
+        </div>
+            <!-- About Section
+    <section class="about-sect"> -->
+        <div class="about">
```
Comment (zieeco) [inline index.html:169]: - Please, kindly use a section tag for your about section and it shouldn't be wrapped inside the works section as this is another section. please fix this.

Comment (zieeco) [reply index.html:169]: *FONT SIZE*

- Please, kindly improve the font size, and weight of **all your text** as this is making your web page hard to read its content as it doesn't improve the user experience.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @canon20paul,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ghost — APPROVED** [review]
Hi @canon20paul , I'm [frank](@frank1738) Your Code Reviewer 

Good work implementing changes requested by the previous reviewer. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

thumbs-up-agree


## Highlights

✅  Great work using the right GitHub Flow.
✅ No Linter errors

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL] I would suggest that you provide working social links on your Readme file

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**